website/docs/r/capacity_reservation_group.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "Compute"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_capacity_reservation_group"
+description: |-
+  Manages a Capacity Reservation Group.
+---
+
+# azurerm_capacity_reservation_group
+
+Manages a Capacity Reservation Group.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example"
+  location = "West Europe"
+}
+
+resource "azurerm_capacity_reservation_group" "example" {
+  name                = "example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  zones               = ["1", "2", "3"]
+  tags = {
+    key = "value1"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `name` - (Required) The name which should be used for this Capacity Reservation Group. Changing this forces a new Capacity Reservation Group to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the  Capacity Reservation Group exists.
```

Comment:
`Group where the Capacity`, there's an extra space between `the` and `Capacity`.